packages/block/test/from-rpc.spec.ts

```diff
@@ -1,3 +1,4 @@
+// cspell:ignore wtxns wtxs wotxns
```

Comment:
Not a big fan of having all these new `cspell:ignore` lines at the beginning of files just for spell checking. Can't we add all of these terms to the config file?

Reply:
Renaming the variables (wtxns -> withTxns) may be the best solution if the variable names are not very intuitive. Otherwise, it would make sense to ignore by file or line, but then we'd have to keep aware of what unique variable names are coming up in frequency of usage, so that we could eventually switch over to adding them to the config file instead of ignoring them on a per line or per file basis.

Reply:
Have addressed -> https://github.com/ethereumjs/ethereumjs-monorepo/pull/3590#issuecomment-2294846162